netutil.py

```diff
@@ -20,10 +20,20 @@ def __init__(self, nic_dict):
         self.driver = "%s (%s)" % (nic_dict.get("Driver", ""),
                                    nic_dict.get("Driver version", ""))
         self.smbioslabel = nic_dict.get("SMBIOS Label", "")
+        self.bond_mode, self.bond_members = None, None
 
     def __repr__(self):
         return "<NIC: %s (%s)>" % (self.name, self.hwaddr)
 
+    @classmethod
+    def create_bonding(cls, devname, mac, bond_mode, bond_members):
+        nic = NIC({"Kernel name": devname,
+                   "Assigned MAC": mac,
+                   })
+        nic.bond_mode = bond_mode
+        nic.bond_members = bond_members
+        return nic
+
```

Comment:
You don't use `cls` why not make a `@staticmethod` then?

Reply:
right, but this is going away anyawy